File: java/src/main/java/token.java
class token{
	
	private String type;
	private String content;

	public token(String type, String content){

		this.type = type;
		this.content = content;

	}

	public String get_type(){

		return this.type;

	}

	public String get_content(){

		return this.content;

	}

}

File: c/HEADER/token.c
#include <stdlib.h>
#include "token.h"

//creating a new token
token* new_token(char *type, char *content){

	token *t = calloc(1, sizeof(token));
	
	t->next_token = NULL;
	t->prev_token = NULL;
    t->type = type;
    t->content = content;
    
	return t;
}

